src/components/OAuth/Facebook/FacebookApi.ts

```diff
@@ -0,0 +1,49 @@
+import { AxiosError } from "axios";
+
+import OAuthApi from "../OAuthApi";
+import { EndpointsEnum, api } from "@/src/axios";
+import { OAuthApiArgs } from "../OAuth.type";
+
+class FacebookApi extends OAuthApi {
+  protected static async facebookApi(facebookAccessToken: string | undefined) {
```

Comment:
I you don`t inherit from FacebookApi  you can change public instead protected

Reply:
if i make this method public it will be accessible outside the class which is not necessary, it might be better to make it private instead of protected

Reply:
indeed,  it will better